Rakefile

```diff
@@ -10,4 +10,12 @@ Rake::TestTask.new(:test) do |t|
   t.warning = false
 end
 
-task :default => :test
+Rake::TestTask.new(:test_deflate) do |t|
+  t.libs << "test"
+  t.libs << "lib"
+  t.test_files = FileList["test/rack_deflate_test.rb"]
+  t.verbose = false
+  t.warning = false
+end
+
+task :default => [:test, :test_deflate]
```

Comment:
I might have gone for this so you would not have to make a change in _test

```ruby
namespace :test do
  desc "Test with Rack::Deflater."
  task(:deflate) do
    ENV['RACK_DEFLATE_ENABLED'] = '1'
    Rake::Task['test'].execute
  end
end
```

Reply:
Talked offline. tl;dr - This will run the whole test suite which would currently fail a few assertions due to the tests expecting a plain text/html response instead of gzip response. Instead of adding `skip`s for that scenario `test_deflate` would allow us to run just a single test file with `Rack::Deflate` middleware enabled.